plugins/Referrers/angularjs/campaign-builder/campaign-builder.controller.js

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Piwik - free/libre analytics platform
```

Comment:
outdated header. but guess we may need to update all files at once one day...

Reply:
yeah at some point we need to  update them all... will update it here anyway 👍